src/frontend/src/lib/components/tokens/SendTokensAmount.svelte

```diff
@@ -18,5 +18,20 @@
 
 	<p>
 		{#if nonNullish(token)}<span>{formatICP(token.toE8s())} <small>ICP</small></span>{/if}
+				{#if nonNullish($icpToUsd)}
```

Comment:
What about extending the check with `{#if nonNullish($icpToUsd) && nonNullish(token)}`?

This way we do not display zero when the token is not defined.